catalogue/webapp/components/ArchiveBreadcrumb/ArchiveBreadcrumb.tsx

```diff
@@ -1,14 +1,14 @@
 import { Work } from '@weco/common/model/catalogue';
 import styled from 'styled-components';
-import DropdownButton from '../DropdownButton/DropdownButton';
-import Icon from '../Icon/Icon';
+import DropdownButton from '@weco/common/views/components/DropdownButton/DropdownButton';
+import Icon from '@weco/common/views/components/Icon/Icon';
 import WorkTitle from '@weco/common/views/components/WorkTitle/WorkTitle';
```

Comment:
Could we move this one too?

Reply:
Yeah, but I'll do that separately. This bit works, let's shake out some more stuff now.